http-gateway/web/src/common/services/init-services.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react'
+
+import { WSManager } from './ws-manager'
+
+import { thingsWSClient } from '@/containers/things/websockets'
+
+export const InitServices = () => {
+  useEffect(() => {
+    // Register the default WS instances
+    if (!WSManager.isInitialized) {
+      WSManager.addWsClient(thingsWSClient)
+      WSManager.registerWSClients()
+    }
+
+    return () => {
+      // WSManager.unregisterWSClients()

```

Comment:
This is commented off, is it handled somewhere else?

Reply:
I might handle it in the future, for now it doesn't need to be, the reconnect logic takes care of it.